bin/checkVersion.js

```diff
@@ -9,6 +9,14 @@ const version = config.engines.node.match(/[0-9.]+/).toString();
 var min = minVersion.split('.');
 var ver = version.split('.');
 
+// turn elements in array from string to number
+min = min.map(function(value, index) {
```

Comment:
These maps could be done on the same line as `min` is originally defined i.e.
```js
var min = minVersion.split('.').map(v => Number(v));
```

Reply:
Should I keep it as is, or make the change?

Reply:
Nah. I'd say leave it as is and go ahead and merge it @mugwump303